src/pages/countries.test.js

```diff
@@ -77,71 +78,31 @@ const mocks = [
               },
             ],
           },
-          {
-            id: "Q781",
-            name: "Antigua and Barbuda",
-            vatRate: null,
-            languages: [
-              {
-                id: "Q1860",
-                name: "English",
-              },
-            ],
-          },
-          {
-            id: "Q414",
-            name: "Argentina",
-            vatRate: 21,
-            languages: [
-              {
-                id: "Q1321",
-                name: "Spanish",
-              },
-            ],
-          },
-          {
-            id: "Q399",
-            name: "Armenia",
-            vatRate: 20,
-            languages: [],
-          },
-          {
-            id: "Q408",
-            name: "Australia",
-            vatRate: 10,
-            languages: [
-              {
-                id: "Q1860",
-                name: "English",
-              },
-            ],
-          },
-          {
-            id: "Q40",
-            name: "Austria",
-            vatRate: 20,
-            languages: [
-              {
-                id: "Q188",
-                name: "German",
-              },
-            ],
-          },
         ],
       },
     },
   },
 ];
 
-it("renders without error", () => {
-  const component = TestRenderer.create(
-    <MemoryRouter >
-      <MockedProvider mocks={mocks} addTypename={false}>
-        <Countries />
-      </MockedProvider>
-    </MemoryRouter>
+const componentToRender = () =>
+  render(
+    <MockedProvider mocks={mocks} addTypename={false}>
+      <MemoryRouter initialEntries={["/countries"]}>
+        <Route path="/countries">
+          <Countries />
+        </Route>
+      </MemoryRouter>
+    </MockedProvider>
   );
+afterEach(cleanup);
+
+it("loading state of apollo", async () => {
+  const { getByText } = componentToRender();
+  await waitFor(() => getByText(/Loading.../i));
+});
 
-  const tree = component.toJSON();
-  expect(tree.children).toContain("Loading...");
+it("Check if loaded properly", async () => {
+  const { getByText } = componentToRender();
+  await waitFor(() => getByText(/Loading.../i));
```

Comment:
Please add data-testid to all of the elements and get it by `getByTestId` or `getAllByTestId` depending on number of elements, this is because classes might be changed in the future or texts inside the divs or whatever, but data-testid attribute usually once set is not changed cause this is used only for testing purpose

Reply:
fixed in last commit! :)